libbeat/outputs/logstash/async.go

```diff
@@ -183,7 +175,33 @@ func (r *msgRef) callback(seq uint32, err error) {
 	}
 }
 
+func newMsgRef(
+	client *asyncClient,
+	data []outputs.Data,
+	cb func([]outputs.Data, error),
+) *msgRef {
+	r := &msgRef{
+		client:    client,
+		count:     1,
+		batch:     data,
+		batchSize: len(data),
+		win:       client.win,
+		cb:        cb,
+		err:       nil,
+	}
+
+	debug("msgref(%p) new: batch=%p, cb=%p", r, &r.batch[0], cb)
```

Comment:
This will create a lot of debug messages. I know that this info is for debugging, but I start to get the feeling we need something in between `info` and `debug`, something like `info --verbose` :-)

Reply:
Yeah, some extra-debug would be nice to have. Normally the debug messages create about 4 new debug messages per batch (2048) events. Not too bad, but super helpful to identify potential issues, should we still face some problems.